src/components/Receiving/ReceivingList.js

```diff
@@ -0,0 +1,152 @@
+import React, { Component } from 'react';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+import ReactRouterPropTypes from 'react-router-prop-types';
+
+import {
+  get,
+  uniqBy,
+} from 'lodash';
+
+import {
+  Button,
+  IconButton,
+  MultiColumnList,
+  Pane,
+  PaneMenu,
+  Paneset,
+  Row,
+  SearchField,
+  TextField,
+} from '@folio/stripes/components';
+
+import { RECEIVING_API } from '../Utils/api';
+import FolioFormattedTime from '../FolioFormattedTime';
+import css from './ReceivingList.css';
+
+class ReceivingList extends Component {
+  static manifest = Object.freeze({
+    query: {
+      initialValue: {
+        query: '',
+      },
+    },
+    receiving_history: {
+      type: 'okapi',
+      path: RECEIVING_API,
+      throwErrors: false,
+      GET: {
+        params: {
+          query: 'purchase_order_id==:{id}',
+        },
+      },
+    },
+  })
+
+  static propTypes = {
+    resources: PropTypes.object,
+    location: ReactRouterPropTypes.location.isRequired,
+    mutator: PropTypes.shape({
+      query: PropTypes.shape({
+        update: PropTypes.func.isRequired,
+      }),
+    }).isRequired,
+  }
+
+  onCloseReceiving = () => {
+    const { location, mutator } = this.props;
+
+    mutator.query.update({
+      _path: location.pathname.replace('/receiving', ''),
+    });
+  }
+
+  getFirstMenu = () => (
+    <PaneMenu>
+      <FormattedMessage id="ui-orders.buttons.line.close">
+        {(title) => (
+          <IconButton
+            aria-label={<FormattedMessage id="ui-orders.buttons.line.closeDialog" />}
+            icon="times"
+            id="clickable-close-new-line-dialog"
+            onClick={this.onCloseReceiving}
+            title={title}
+          />
+        )}
+      </FormattedMessage>
+    </PaneMenu>
+  );
+
+  render() {
+    const { resources } = this.props;
+    const receivingList = get(resources, ['receiving_history', 'records', 0, 'receiving_history'], []);
```

Comment:
usually it's handy to specify attribute `records: 'receiving_history',` in resource's manifest